packages/rfw/test/material_widgets_test.dart

```diff
@@ -123,6 +123,28 @@ void main() {
                 ),
               ),
             ),
+            Divider(),
+            Padding(
+              padding: [20.0],
+              child: Center(
+                child: DropdownButton(
+                  value: 'foo',
+                  items: [
+                    {
+                      value: 'foo',
+                      child: Text(text: 'foo'),
+                      onTap: event 'menu_item' {},
+                    },
+                    {
+                      value: 'bar',
+                      child: Text(text: 'bar'),
+                      onTap: event 'menu_item' {},
+                    },
+                  ],
+                  onChanged: event 'dropdown' {},
+                ),
+              ),
+            ),
```

Comment:
would be good to also test:
 - more of the parameters that you have implemented
 - different types of values
 - that the widget actually works interactively correctly as we'd expect

Reply:
Done!